perllib/Dataset/UK/Stats19/Cmd/UpdateParticipants.pm

```diff
@@ -0,0 +1,59 @@
+package Dataset::UK::Stats19::Cmd::UpdateParticipants;
+use Moo;
+use MooX::Cmd;
+use Path::Tiny;
+use Text::CSV_XS;
+use File::BOM;
+use List::MoreUtils 'zip';
+use List::Util 'first';
+use mySociety::MaPit;
+use feature 'say';
+
+has cobrand => (
+    is => 'lazy',
+    default => sub { FixMyStreet::Cobrand::Smidsy->new },
+);
+
+use FixMyStreet;
+use FixMyStreet::App;
+
+sub execute {
+    my ($self, $args, $chain) = @_;
+    my ($stats19) = @{ $chain };
+
+    my $db = $stats19->db;
+
+    my $bike = $db->resultset('VehicleType')->find({ label => 'Pedal cycle' });
+
+    my $vehicles = $db->resultset('Vehicle')->search(
+        { vehicle_type_code => $bike->code },
+        { group_by => 'me.accident_index',
+          prefetch => 'accident' },
+    );
+
+    my $problem_rs = FixMyStreet::App->model('DB::Problem');
```

Comment:
This should be FixMyStreet::DB->resultset("Problem"), no need to load the whole app.